### PR TITLE
auth view

### PR DESCRIPTION
**Pull request by wowKimsohyun17**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/31022)

## WHAT this PR does?
auth view

## Notes Impacted Areas:
- N/A.

## Attachments:
Screenshot from 2020-10-26 08-56-41
Screenshot from 2020-10-26 08-56-45

### CODE REVIEW

**Comment by wowKimsohyun17** [comment]
ready

**Comment by wowKimsohyun17** [comment]
ready

File: resources/views/auth/passwords/confirm.blade.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">{{ __('Confirm Password') }}</div>
+
+                <div class="card-body">
+                    {{ __('Please confirm your password before continuing.') }}
+
+                    <form method="POST" action="{{ route('password.confirm') }}">
+                        @csrf
```
Comment (huyenttm-0901) [inline resources/views/auth/passwords/confirm.blade.php]: dùng lang hoặc trans

File: resources/views/auth/passwords/confirm.blade.php [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                                <button type="submit" class="btn btn-primary">
+                                    {{ __('Confirm Password') }}
+                                </button>
+
+                                @if (Route::has('password.request'))
+                                    <a class="btn btn-link" href="{{ route('password.request') }}">
+                                        {{ __('Forgot Your Password?') }}
+                                    </a>
+                                @endif
+                            </div>
```
Comment (huyenttm-0901) [inline resources/views/auth/passwords/confirm.blade.php]: tương tự

**Review by huyenttm-0901 — COMMENTED** [review]
sửa hết i18n

**Comment by wowKimsohyun17** [comment]
ready

**Pull request merged**